source/grammar/openpulseLexer.g4

```diff
@@ -0,0 +1,7 @@
+lexer grammar openpulseLexer;
+import qasm3Lexer;
+
+CAL: 'cal';
```

Comment:
This should be parsed by the OQ 3 parser, not the OpenPulse one - it's a bug that the `cal` statement is missing from there.  I've opened #368 to track that.